core/src/main/java/com/linecorp/armeria/client/Endpoint.java

```diff
@@ -180,13 +185,17 @@ private static String removeUserInfo(String authority) {
     private final String authority;
     private final String strVal;
 
+    @Nullable
+    private final DefaultAttributeMap attributes;
```

Comment:
Q) If only `attributes` changes, the change won't be reflected in `DynamicEndpointGroup`

https://github.com/line/armeria/blob/b34ae5975a089c33ead3a83c2ff5d60b015934af/core/src/main/java/com/linecorp/armeria/client/endpoint/DynamicEndpointGroup.java#L191

Is there no need to also check if attributes have changed?

Reply:
That's a good question. I guess we should. What do you think, @eisig and @ikhoon ?

Reply:
Agreed. `DynamcEndpointGroup` should see the updated attributes.